lib/editor/actions/editor.js

```diff
@@ -513,6 +513,10 @@ export function fetchBaseGtfs ({
             # Fetch dates to ensure we can do validation in the UI
             # (avoid duplicate dates).
             dates
+            # Fetch exemplar for display of exception based service in calendar select
+            exemplar
```

Comment:
Why is this being fetched from the server?

Reply:
We need the exemplar to determine in the Timetable if the exception represents a calendar